servers/src/main/java/tachyon/master/InodeFile.java

```diff
@@ -97,6 +109,23 @@ public InodeFile(String name, int id, int parentId, long blockSizeByte, long cre
   }
 
   /**
+   * Create a new InodeFile.
+   *
+   * @param name The name of the file
+   * @param id The inode id of the file
+   * @param parentId The inode id of the parent of the file
+   * @param blockSizeByte The block size of the file, in bytes
+   * @param creationTimeMs The creation time of the file, in milliseconds
+   * @param permissionStatus the file permission information
+   */
+  public InodeFile(String name, int id, int parentId, long blockSizeByte, long creationTimeMs,
```

Comment:
I think using a new construct method is a better idea. Because it doesn't need to change a lot of original code.